src/Stratis.Bitcoin.Features.BlockStore/BlockStoreFeature.cs

```diff
@@ -146,6 +151,8 @@ public override async Task InitializeAsync()
                 this.connectionManager.Parameters.Services |= NetworkPeerServices.NODE_WITNESS;
 
             this.blockStoreSignaled.Initialize();
+
+            this.addressIndexer.Initialize();
```

Comment:
I stil think we should consider two flags `-txindex` and `-addrindex` this component may be heavy

Reply:
yeah, I have it in my TODO list for the next PR

Reply:
this component is pretty big so I'm doing it in small chunks. there will be at least 2 more PRs

Reply:
>yeah, I have it in my TODO list for the next PR
Excellent, though won't this mess up current running nodes?

Reply:
Maybe you want to disable the indexer for now.

Reply:
Is it not disabled by default?

Reply:
It is disabled by default.